source/_patterns/02-organisms/components/hero-banner.json

```diff
@@ -0,0 +1,33 @@
+{
+  "image": {
+    "fallback": {
+      "defaultPath": "https://unsplash.it/1114/336",
+      "altText": ""
+    }
+  },
+  "subjects": {
+    "list": [
+      {
+        "name": "Structure Biology and Mulecular Biophysics",
+        "url": "#"
+      }
+    ]
+  },
+  "title": "Filamentation and the regulation of PRPP synthase",
+  "titleLength": "small",
+  "url": "#",
+  "summary": "Cryo-EM structures of two types of prokaryotic Phosphoribosyl pyrophosphate synthase filaments reveal tat filamentation into cytoophidia provides a new layer of metabolic regulation.",
+  "authors": "Lee R Berger, John Hawks",
+  "meta": {
+    "url": "#",
+    "text": "Research article",
+    "date": {
+      "forHuman": {
+        "dayOfMonth": 23,
+        "month": "Jan",
+        "year": 2022
+      },
+      "forMachine": "2022-01-23"
+    }
+  }
+}
```

Comment:
The default json should only contain the required values and should encourage use of the variant examples. See: https://github.com/elifesciences/pattern-library/blob/develop/source/_patterns/00-atoms/components/author-details.json

Reply:
The default json data is changed to cotains only reuqired fields